src/UI/Internal/Utils/Element.elm

```diff
@@ -56,29 +58,31 @@ id value =
         |> Element.htmlAttribute
 
 
-overlayBackground : msg -> Element msg
-overlayBackground onClickMsg =
+clickElsewhereToLeave : msg -> List (Attribute msg) -> Element msg -> Element msg
+clickElsewhereToLeave onClickMsg backgroundStyleAttributes foregroundContent =
```

Comment:
TBH I prefer the old name, this name gives me the impression that it should be of type `Attribute msg` instead of an element

Reply:
What do you think of "transparentOverlay" ?